File: Java_Assignment/Assignment_59/programAQ3.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement59Question3.java
//  Description : Write java program to print below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

     * * * * *
      * * * *
       * * *
        * *
         *
*/

public class programAQ3
{
    public static void main(String args[])
    {
        int no = 5;

        int i = 0, j = 0;

        for(i = 1;i <= no;i++)
        {
            for(j = 1;j <= i - 1;j++)
            {
                System.out.print(" ");
            }

            for(j = 1; j <= no - i + 1;j++)
            {
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}

File: C_Assignment/Assignment_40/programAQ3.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement40Question3.c
//  Description : Write a recursive program which display below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Output : 5 4 3 2 1
Prototype :

*/

#include<stdio.h>

void Display()
{
    static int i = 5;

    if(i >= 1)
    {
        printf("%d\t",i);
        i--;
        Display();
    }
}

int main()
{
   Display();

    return 0;
}